src/systems/ec/implementations/react/utilities/background/stories/Index.jsx

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import classnames from 'classnames';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, select } from '@storybook/addon-knobs';
+
+storiesOf('Utilities|Background', module)
+  .addDecorator(withKnobs)
+  .add('custom', () => {
+    const background = select(
+      'Background colour (sample)',
+      {
+        Default: 'ecl-u-bg-default',
+        Primary: 'ecl-u-bg-primary ecl-u-type-colour-white-100',
```

Comment:
there's something wrong in the way we name our colours (it's not specific to this PR)

there's no primary colour (singular), there are 3 primary colours

there's no secondary colour, no shade colour

how do we fix this?

Reply:
Here is me suggestion:
we use the colours as they are on the guidelines, with the exact same names.
For now, we don't have any use of semantic color names